src/DotVVM.Framework/Compilation/BindingRequiredResourceVisitor.cs

```diff
@@ -16,9 +16,9 @@ namespace DotVVM.Framework.Compilation
     public class BindingRequiredResourceVisitor : ResolvedControlTreeVisitor
     {
         private readonly ControlResolverMetadata requiredResourceConrolMetadata;
-        public BindingRequiredResourceVisitor(ControlResolverMetadata requiredResourceConrolMetadata)
+        public BindingRequiredResourceVisitor(ControlResolverMetadata requiredResourceControlMetadata)
```

Comment:
This is not going to be used, fine to merge.

Reply:
Do you me want to to add this (and similar cases) to the main PR?